routes/api/chores.js

```diff
@@ -27,3 +27,221 @@ router.get('/:group_ID', (req, res) => {
         })
 });
 
+// Route        POST api/chores/
+// Description  Adds a chore to the group. 
+// Access       Public I think
```

Comment:
Please add a comment with the parameters for this endpoint here (chore_assigned_user, chore_assigned_user_index, chore(Please rename this one), chore_name)